### ISSUE TITLE
use CAUS as auto scaler

### ISSUE THREAD

**Issue by delvh** [t2-project/kube#10]
Description TBA
(only possible once CAUS is done)

**Comment by delvh** [comment]
Still on hold as the original CAUS uses deprecated/ removed APIs,
and the new CAUS(es) are only a proof of concept.

**Comment by davidkopp** [comment]
A good alternative to CAUS could be [KEDA](https://keda.sh/).

I don't know much about CAUS, but at least the feature of autoscaling based on the rate messages arrive in a queue is also supported by KEDA (see e.g. mode `MessageRate` in the scaler [RabbitMQ Queue](https://keda.sh/docs/2.13/scalers/rabbitmq-queue/)).
